src/main/java/org/variantsync/diffdetective/variation/tree/VariationNode.java

```diff
@@ -437,7 +450,7 @@ public VariationTreeNode toVariationTree() {
             getNodeType(),
             getFormula().clone(),
             getLineRange(),
-            new ArrayList<String>(getLabelLines())
+            getLabel().lines()
```

Comment:
Why remove the copying? We still clone te formula.

Reply:
Good catch. That a mistake.